book/run/troubleshooting.md

```diff
@@ -2,8 +2,42 @@
 
 This page tries to answer how to deal with the most popular issues.
 
+- [Troubleshooting](#troubleshooting)
+  - [Database](#database)
+    - [Docker](#docker)
+    - [Error code 13](#error-code-13)
+    - [Slow database inserts and updates](#slow-database-inserts-and-updates)
+      - [Compact the database](#compact-the-database)
+      - [Re-sync from scratch](#re-sync-from-scratch)
+    - [Database write error](#database-write-error)
+    - [Concurrent database access error (using containers/Docker)](#concurrent-database-access-error-using-containersdocker)
+  - [Hardware Performance Testing](#hardware-performance-testing)
+    - [Disk Speed Testing with IOzone](#disk-speed-testing-with-iozone)
+
+
 ## Database
 
+### Docker 
+
+Externally accessing a `datadir` inside a named docker volume will usually come with folder/file ownership/permissions issues.
+
+**It is not recommended** to use the path to the named volume as it will trigger an error code 13.
+
+Example: `db-access --path /var/lib/docker/volumes/named_volume/_data/eth/db`
+
+### Error code 13 
+
+`the environment opened in read-only code: 13`
```

Comment:
should we map this error code to a new error variant that displays this recommendation?